### DISCUSSION TITLE
How do I train Alpaca on a different dataset

### DISCUSSION THREAD

**Discussion by Aityz** [tloen/alpaca-lora#528] [General]
Now that I have trained Alpaca (using this repo), can I use it to train on another dataset (for example open-instruct-v1), or do I need a different repository?

**Comment by Oyiyi** [comment]
Just change the dataset path to your new dataset into hf or your local json file.
python finetune.py \
    --base_model '' \
    --data_path 'dataset_path_here' \
    --output_dir './lora-alpaca-finetuned-outputs'
